README.md

```diff
@@ -754,10 +754,17 @@ Place a cancellation order for all pending orders in the provided account.
 Information about a security on the WealthSimple Trade Platform.
 
 
-| Parameters|Required|
-|----------|---------------------|
-| tokens |Yes|
-| ticker |Yes|
+| Parameters|Required|Notes|
+|----------|---------------------|-|
+| tokens |Yes| |
+| ticker |Yes| The exchange the security trades on may be included after a colon. For example, `RMD:NYSE`|
```

Comment:
It would be helpful to add the yahoo link you specified in your issue to indicate to users what the ids for the exchanges are.

Reply:
The exchange suffixes here are `:{EXCHANGE_NAME}`, whereas the exchange suffixes in that link are different, for example `.TO` for the TSX.

Reply:
Fair enough. Since WS Trade only supports three exchanges, i think it would be easy enough to just list out all possible exchange suffixes instead of confusing the user with the ‘.’ separator in the link.

Reply:
For clarity and quick retrieval purposes, please add above the Parameters table something like this:

--

**Specifying an exchange**
You are allowed to specify the exchange for securities as part of the `ticker` parameter through a `:` postfixing format. The supported exchanges ids are:

* **NASDAQ** (NASDAQ exchange)
* **TSX** (Toronto Stock exchange)
* **NYSE** (New York Stock exchange)

For example, `PSI` trades on the NYSE and TSX stock exchanges. If you wish to retrieve the TSX stock, specify the ticker as `PSI:TSX`. Otherwise, specify the ticker as `PSI:NYSE`.

--

This way, all supported exchanges and their ids are communicated to the users.